packages/pds/src/app-view/api/app/bsky/feed/getAuthorFeed.ts

```diff
@@ -38,17 +39,37 @@ export default function (server: Server, ctx: AppContext) {
       const feedService = ctx.services.appView.feed(ctx.db)
       const graphService = ctx.services.appView.graph(ctx.db)
 
+      // maybe resolve did first
```

Comment:
there's a method on actorService for this that may be nice to use: `getActorDid`